overlay.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Overlay</title>
+    <link rel="stylesheet" type="text/css" href="over.css">
+</head>
+<body>
+    <div id="top">
+        <img src="https://openclipart.org/download/215361/shadowed-news-icon.svg">
```

Comment:
From "openclipart" it sounds like this is fine, but make sure you've confirmed this is okay to use without any attribution.

Reply:
this is just a filler image for now! currently trying to design a new keeptabs logo